gpu-descriptor/src/allocator.rs

```diff
@@ -114,15 +114,18 @@ struct DescriptorBucket<P> {
 
 impl<P> Drop for DescriptorBucket<P> {
     fn drop(&mut self) {
-        assert_eq!(
-            self.total, 0,
-            "Allocator dropped before all sets were deallocated"
-        );
+        #[cfg(feature = "std")]
```

Comment:
So without `"std"` feature it won't do anything. Is that best we can do here?

Reply:
I dunno. Why would anybody use gpu-descriptor without `std`?

Reply:
I dunno, to render something on their microwave LCD :)
I just though that since crate itself does not need `std`, supporting `no_std` wouldn't hurt.